package.json

```diff
@@ -39,5 +41,10 @@
   "bugs": {
     "url": "https://github.com/mafintosh/hyperdb/issues"
   },
-  "homepage": "https://github.com/mafintosh/hyperdb#readme"
+  "homepage": "https://github.com/mafintosh/hyperdb#readme",
+  "standard": {
+    "ignore": [
+      "test/helpers/fuzzing.js"
+    ]
```

Comment:
is this needed?

Reply:
Well the `eval` calls both produce a warning and mess with the static analysis (it doesn't see what we reference `put` inside the eval'd function). If there's a way to do this without eval then nope.

Reply:
Use new Function instead of eval?